brave/brave-core
# Fix Brave Ads duplicate feature names

`IdleDetection` feature is renamed to `UserIdleDetection` because it has the same name as chromium feature.


Resolves https://github.com/brave/brave-browser/issues/29263



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Check that `idle_time_threshold` parameter of `UserIdleDetection` feature controls user idle time threshold.
- Check that `notification_ad_timeout` parameter of `NotificationAds` feature is working
- Check that custom ad notification is shown when `CustomNotificationAds` feature enabled
- On Windows or on MacOS: Check that custom ad notification becomes attached if `should_attach_to_browser_window` parameter is specified
